### ISSUE TITLE
Nullish coalescing, Issue

### ISSUE THREAD

**Issue by careless10** [coinbase/rest-hooks#902]
**React version**
16.13.1

**Concurrent mode**
no

**Rest Hooks version**
5.1.3

**Describe the bug**
```javascript
./node_modules/@rest-hooks/core/lib/state/NetworkManager.js 104:167
Module parse failed: Unexpected token (104:167)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| 
|       dispatch(createReceive(data, { ...action.meta,
>         dataExpiryLength: ((_action$meta$options = action.meta.options) === null || _action$meta$options === void 0 ? void 0 : _action$meta$options.dataExpiryLength) ?? this.dataExpiryLength
|       }));
|       return data;
```

**To Reproduce**
Upgrading from 4.5 -> 5.1.3
On a CRA project with (react-scripts:4.0.3)

**Expected behavior**
As Nullish coalescing is already working on my project i expect this to pass, but im not sure about how babel is handling this

**Additional Context**
The babel-loader that the issue is referencing is at version 8.1.0,
Compiling core with, "yarn run prepare" using the babel config, (Added Nullish coalescing plugin)
```javascript
module.exports = function (api) {
  api.cache.using(() => process.env.NODE_ENV);
  return {
    presets: [
      [
        '@anansi/babel-preset',
        {
          typing: 'typescript',
          loose: true,
        },
      ],
    ],
  plugins: ["@babel/plugin-proposal-nullish-coalescing-operator"],
    // allows us to load .babelrc in addition to this
    babelrcRoots: ['packages/*', '__tests__'],
  };
};
```
Makes the compiled folder 'lib' content works.

Thanks for your time,
Thank you for this beautiful package aswell👍🏽

**Comment by ntucker** [comment]
Thanks for the report! It's super clear so we should be able to resolve soon.

**Comment by ntucker** [comment]
Wow, I just realized how crazy it is that shipped features for years in browsers and CRA can't even parse the files. I'm gonna try fixing the target for stuff in "module" so rather than being reasonably old it just targets a standard that even non-updating libraries like CRA will be able to read.

**Comment by ntucker** [comment]
Just released 5.1.4 with the fix. Verify if it works now?

**Comment by careless10** [comment]
Yep, like a charm, Thank you
For anyone facing this issue, if upgrading is not working please delete the package from node_modules
and reinstall it again
